src/App.tsx

```diff
@@ -1,19 +1,125 @@
+/* eslint-disable react/no-unused-state */
 import React from 'react';
-import './App.css';
-
+import './App.scss';
+import todos from './api/todos';
 import users from './api/users';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types';
+
+const preparedTodos: Todo[] = todos.map((todo) => {
+  return {
+    ...todo,
+    user: users.find(user => user.id === todo.userId) || null,
+  };
+});
+
+interface State {
+  todos: Todo[];
+  todoTitle: string;
+  userId: number;
+}
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    todos: [...preparedTodos],
+    todoTitle: '',
+    userId: 0,
+  };
+
+  handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    event.preventDefault();
+
+    const text = event.target.value;
+
+    this.setState({
+      todoTitle: text,
+    });
+  };
+
+  handleUserChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    event.preventDefault();
+
+    const userId = +event.target.value;
+
+    this.setState({
+      userId,
+    });
+  };
+
+  handleSubmit = () => {
+    const { userId, todoTitle } = this.state;
+
+    if (!todoTitle) {
+      return;
+    }
+
+    const nextID = Math.max(...todos.map(todo => todo.id)) + 1;
+
+    const currentUser = users.find(user => user.id === userId);
```

Comment:
Check if this is undefined, and if it is, you shouldn't create a user, that way you won't need to hardcode 0 or 1 as id